Makefile

```diff
@@ -92,9 +95,13 @@ gen/version.c: *.c *.h gen/.build
 		echo '"redsocks.git/'`git describe --tags`' $(CRYPTO)"'; \
 		if [ `git status --porcelain | grep -v -c '^??'` != 0 ]; then \
 			echo '"-unclean"'; \
-		fi \
+		fi; \
+		echo '"\n"'; \
```

Comment:
With this strange line, I got `missing terminating " character` error when compile

Reply:
Thanks for reporting. I pushed commit a40e60b48438e2ca1bcf6112a64431da6951c97a to fix this error.